.gitignore

```diff
@@ -99,7 +99,7 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
+poetry.lock
```

Comment:
I see you added `poetry.lock` file to git but it's also set to be ignored here. Any reason why?

Reply:
The poetry.lock file was already present and is generated automatically by Poetry. I decided to remove it and add it to .gitignore, as it will be regenerated when running Poetry. It doesn't seem necessary to track it in version control, but please let me know if I should keep it as is.